### PR TITLE
[CIAPP-6905] Adding CI Visibility audit events

### PR DESCRIPTION
**Pull request by rodrimendoza**
### What does this PR do?
[Jira](https://datadoghq.atlassian.net/browse/CIAPP-6905)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by andrea-mosk — APPROVED** [review]
Could you change the link for ingestion controls guide as well?

File: content/en/account_management/audit_trail/events.md [hunk 1]
```diff
@@ -109,9 +109,13 @@ See the [Audit Trail documentation][2] for more information on setting up and co
...
+|---------------------------------|--------------------------------------------------------------|-----------------------------------------------------------------------|
+| [Repository Default Branch][30] | A user modified the default branch of a repository.          | `@evt.name:"CI Visibility" @asset.type:ci_app_repository`             |
+| [Test Service Settings][92]     | A user created or modified the settings of a Test Service.   | `@evt.name:"CI Visibility" @asset.type:ci_app_test_service_settings`  |
+| [GitHub Account Settings][93]   | A user has modified the GitHub Account Settings.             | `@evt.name:"CI Visibility" @asset.type:github_opt_ins`                |
+| [Exclusion filters][94]         | The exclusion filters have been modified.                    | `@evt.name:"CI Visibility" @asset.type:ci_app_exclusion_filters`      |
+| [Quality Gates Rule][95]        | A user has created, modified or deleted a Quality Gate Rule. | `@evt.name:"CI Visibility" @asset.type:ci_app_quality_gates_rule`     |
```
Comment (urseberry) [inline content/en/account_management/audit_trail/events.md:118]:
```suggestion
| [Repository default branch][30] | A user modified the default branch of a repository.          | `@evt.name:"CI Visibility" @asset.type:ci_app_repository`             |
| [Test service settings][92]     | A user created or modified the settings of a test service.   | `@evt.name:"CI Visibility" @asset.type:ci_app_test_service_settings`  |
| [GitHub account settings][93]   | A user has modified the GitHub account settings.             | `@evt.name:"CI Visibility" @asset.type:github_opt_ins`                |
| [Exclusion filters][94]         | The exclusion filters have been modified.                    | `@evt.name:"CI Visibility" @asset.type:ci_app_exclusion_filters`      |
| [Quality gates rule][95]        | A user has created, modified or deleted a quality gate rule. | `@evt.name:"CI Visibility" @asset.type:ci_app_quality_gates_rule`     |
```

File: content/en/account_management/audit_trail/events.md [hunk 2]
```diff
@@ -237,9 +241,9 @@ See the [Audit Trail documentation][2] for more information on setting up and co
...
-| [Workflow Schedule][89] | A user created, deleted, or modified a schedule for a workflow. | `@evt.name:"Workflows" @asset.type:workflow_schedule @action:(created OR deleted OR modified)` |   
-| [Workflow Action][90] | A user responded to a Slack prompt during the execution of a workflow. | `@evt.name:"Workflows" @asset.type:workflow_action @action:(responded)` | 
-| [Custom Connection][91] | A user created, deleted, or modified a connection. | `@evt.name:"Custom Connections" @asset.type:custom_connection @action:(created OR deleted OR modified)` | 
+| [Workflow Schedule][89] | A user created, deleted, or modified a schedule for a workflow. | `@evt.name:"Workflows" @asset.type:workflow_schedule @action:(created OR deleted OR modified)` |
+| [Workflow Action][90] | A user responded to a Slack prompt during the execution of a workflow. | `@evt.name:"Workflows" @asset.type:workflow_action @action:(responded)` |
+| [Custom Connection][91] | A user created, deleted, or modified a connection. | `@evt.name:"Custom Connections" @asset.type:custom_connection @action:(created OR deleted OR modified)` |
```
Comment (urseberry) [inline content/en/account_management/audit_trail/events.md:246]:
```suggestion
| [Workflow schedule][89] | A user created, deleted, or modified a schedule for a workflow. | `@evt.name:"Workflows" @asset.type:workflow_schedule @action:(created OR deleted OR modified)` |
| [Workflow action][90] | A user responded to a Slack prompt during the execution of a workflow. | `@evt.name:"Workflows" @asset.type:workflow_action @action:(responded)` |
| [Custom connection][91] | A user created, deleted, or modified a connection. | `@evt.name:"Custom Connections" @asset.type:custom_connection @action:(created OR deleted OR modified)` |
```

**Review by urseberry — COMMENTED** [review]
capitalization nits

File: content/en/account_management/audit_trail/events.md (hunk 2 above)
Comment (rodrimendoza) [reply content/en/account_management/audit_trail/events.md:246]: I'm not sure about this. I'm not even changing anything here, it was just a auto-formatting thing. You say I should change this caps even though these events are from another team?

Comment (rtrieu) [reply content/en/account_management/audit_trail/events.md:247]: Hi @rodrimendoza , yes, would you mind accepting the suggestion so we can keep the style consistent? Thank you!

**Review by anshumgargdd — APPROVED** [review]
Approved from audit trail side.

**Review by urseberry — APPROVED** [review]

**Pull request merged**